redefinirsenha.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE-edge">
+		<meta name="Viewport" content="width=device-width, initial-scale=1.0">
+		<title>Recuperar Senha</title>
+		<link rel="stylesheet" href="styles.css">
```

Comment:
seria ideal criar arquivos para cada um, assim cada página só usa o que for necessário... vale a pena ter a ideia de ter um arquivo base que vai conter as configurações padrões compartilhada entre as páginas

Reply:
Eu meio que crio um "biblioteca" do projeto com as estilizações padrões e uso somente o que eu preciso nas paginas, consequentemente o código fica mais legível e mais fácil o entendimento, até porque é menos código por página.